ats/jobs/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from model_utils import Choices
+from django.utils.translation import ugettext_lazy as _
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=25)
+    description = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = "Category"
+        verbose_name_plural = "Categories"
+
+
+class Job(models.Model):
+    STATUS = Choices(
+        (1, 'DRAFT', _('Draft')),
+        (2, 'ACTIVE', _('Active')),
+        (3, 'SHORT_LISTED', _('Short Listed')),
```

Comment:
You confused between job and JobApplicaton, Job it shouldn't contain `SHORT_LISTED` and `REJECTED`.
This is the status of each user per job, not the status of the job.